nupic/algorithms/anomaly.py

```diff
@@ -100,6 +103,13 @@ def __init__(self, slidingWindowSize = None, mode=MODE_PURE):
       raise ValueError("Invalid anomaly mode; only supported modes are: "
                        "Anomaly.MODE_PURE, Anomaly.MODE_LIKELIHOOD, "
                        "Anomaly.MODE_WEIGHTED; you used: %r" % self._mode)
+    self._binaryThreshold = binaryAnomalyThreshold
+    if binaryAnomalyThreshold is not None and ( 
+          not isinstance(binaryAnomalyThreshold, float) or
+          binaryAnomalyThreshold >= 1.0  or 
```

Comment:
actually, I defined the threshold on open interval (0,1) because 0/1 values would not make much sense: no anomalies/all anomalies.